prettylogger/logger.go

```diff
@@ -103,12 +100,19 @@ func getSymbol(level string) string {
 	case "warning":
 		return "❗ "
 	case "success":
-		return "✔️  "
+		return "✅  "
 	case "fatal", "error":
 		return "❌  "
 	case "debug":
-		return "—  "
+		return "🐞  "
 	default:
-		return "〜 "
+		return "ℹ️ "
```

Comment:
renders no spaces after

<img width="563" alt="Screenshot 2023-08-11 at 3 08 34 PM" src="https://github.com/kubescape/go-logger/assets/132510/25c9f35a-a508-4605-aebe-2f25422202e1">

because it's a double width character and OS X terminal is dumb.

let's see how Minikube does this, and probably put a space before all the emoji to ensure we get this right.